### PR TITLE
Add insurance and create invoice functionality added

### PR DESCRIPTION
**Pull request by prantikdasgupta**

### CODE REVIEW

File: sellstrom.php [hunk 1]
```diff
@@ -806,12 +846,24 @@ public function hookBackofficeHeader($params)
 		$this->processForm($params);
 
 		$content_data = array();
-		$content_data['ps_version'] = _PS_VERSION_;
+		$content_data['ps_version'] = (float)_PS_VERSION_;
```
Comment (Quetzacoalt91) [inline sellstrom.php]: This line is strange by the behaviour you could have in result:
Why don't you replace it with the following one:
```php
$content_data['show_panel'] = version_compare($content_data['ps_version'], '1.5', '>=');
```

File: views/templates/admin/hookbackoffice.tpl [hunk 2]
```diff
@@ -24,305 +24,344 @@
...
-	background-color: #FFBABA;
-}
-.isa_info i, .isa_success i, .isa_warning i, .isa_error i {
-	margin:10px 22px;
-	font-size:2em;
-	vertical-align:middle;
-}
-</style>
+<link href="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/css/sellstrom.css" rel="stylesheet" type="text/css">
+<script type="text/javascript" src="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/js/sellstrom.js"></script>
```
Comment (Quetzacoalt91) [inline views/templates/admin/hookbackoffice.tpl]: Do not include your files this way. Please use instead the functions addCss() and addJs(). Here is an example here:
https://github.com/PrestaShop/gadwords/blob/83d13209fb71bafab9307839cb27c78317dade78/gadwords.php#L66

File: views/templates/admin/hookheader.tpl [hunk 3]
```diff
@@ -24,13 +24,19 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
+<link href="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/css/sellstrom.css" rel="stylesheet" type="text/css">
+<script type="text/javascript" src="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/js/sellstromFrontEnd.js"></script>
```
Comment (Quetzacoalt91) [inline views/templates/admin/hookheader.tpl]: Same comment about the CSS and JS file inclusion.

File: sellstrom.php (hunk 1 above)
Comment (prantikdasgupta) [inline sellstrom.php]: Fixed

File: views/templates/admin/hookbackoffice.tpl (hunk 2 above)
Comment (prantikdasgupta) [inline views/templates/admin/hookbackoffice.tpl]: Fixed

File: views/templates/admin/hookheader.tpl (hunk 3 above)
Comment (prantikdasgupta) [inline views/templates/admin/hookheader.tpl]: Fixed

**Comment by Quetzacoalt91** [comment]
Hi,

Be careful, you moved your images in the properly folder but forgot to change the paths in the code !
In consequence, I get errors on many pages (Carrier list, ajax when adding to cart etc.). See the following screenshot for more details:
capture du 2015-02-27 16 21 49

**Comment by prantikdasgupta** [comment]
Hi Thomas,
There were two places where the change of path was missing. I have fixed it now.
Could you please verify and let me know if it's working fine.

Thanks,
Prantik

**Comment by Quetzacoalt91** [comment]
Hi @prantikdasgupta,

Thanks, I have just merged your PR and the changes should be available from tomorrow.

Regards

**Pull request merged**